README.md

```diff
@@ -1265,6 +1265,7 @@ Table of Contents
   * [Unicorn Platform](https://unicornplatform.com/) - Effortless landing page builder with hosting. 1 website for free.
   * [react-favicon.com](https://react-favicon.com/) - Generate Favicons for your web site using React and JSX using any font and icon library.
   * [svgrepo.com](https://www.svgrepo.com/) - Explore, search and find the best fitting icons or vectors for your projects using wide variety vector library. Download free SVG Vectors for commercial use.
+  * [Windframe](https://devwares.com/windframe/) - Visual builder and editor for rapidly prototyping and building stunning webpages using tailwind css.
```

Comment:
It's not really clear, are these limits monthly or one-off like a trial?

![Screenshot 2022-02-14 at 18 48 54](https://user-images.githubusercontent.com/82342/153918605-92bb938c-adbb-415b-8714-7fc0e32e5245.png)

Reply:
oh sorry about that. It's a one-off for free accounts it does not renew every month.

Reply:
Thanks, then I am afraid thats a free trial not a free tier and so would not be suited to the list.

Thank you!